libs/libc/misc/Kconfig

```diff
@@ -112,3 +112,17 @@ config LIBC_MEM_FD_VFS_PATH
 	depends on !LIBC_MEMFD_ERROR
 	---help---
 		The relative path to where memfd will exist in the tmpfs namespace.
+
+config LIBC_MAX_PATHBUFFER
+	int "Maximum size of a temporary file path buffer array"
+	range 0 32
+	default 2
```

Comment:
So this should either be 0 or only be set if fs_lock is enabled.
Otherwise we're wasting 512 bytes of RAM.

Reply:
actually, we don't need do this type of manual optimization, the linker is enough smart to remove the unused code and data from the final image automatically if nobody calls lib_[get|put]_pathbuffer.